packages/editor/src/components/post-publish-panel/style.scss

```diff
@@ -121,6 +121,8 @@
 		align-items: flex-start;
 		text-wrap: balance; // Fallback for Safari.
 		text-wrap: pretty;
+		display: inline-block;
+		word-break: break-word;
```

Comment:
Where is this being used?

Reply:
You can check these changing in post publish panel, 
<img width="178" alt="Screenshot 2024-06-13 at 7 26 55 PM" src="https://github.com/WordPress/gutenberg/assets/80690679/5fecafa0-8ab1-4a37-a26e-2b8eeaa48c46">

Reply:
Hey @Soean let me know if you need these changes, because it doesn't look that good and seems unnecessary!